benchmarks/encoder/utils.py

```diff
@@ -37,6 +37,7 @@ def gen_traces(config):
     services = _random_values(16, 16)
     tag_keys = _random_values(config.ntags, 16)
     metric_keys = _random_values(config.nmetrics, 16)
+    dd_origin_values = _random_values(16, 16)
```

Comment:
I wonder if we do this vs `("synthetics", "ci-app", ...)`?

probably doesn't matter _too_ much